src/Symfony/Component/HttpKernel/EventListener/TranslatorListener.php

```diff
@@ -24,6 +24,8 @@
  * Synchronizes the locale between the request and the translator.
  *
  * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @deprecated since Symfony 4.2
```

Comment:
this is missing the `@trigger_error`. And it should be 4.3 as we are in feature freeze